### DISCUSSION TITLE
bullet flight direction

### DISCUSSION THREAD

**Discussion by vinurd** [4ian/GDevelop#2336] [Enhancement ideas (and their technical discussions)]
## Description
There is a constant problem with the creation of a bullet flight. Namely, in the direction of flight of the bullet. Addressed to a forum. Dali on the forum, an incredibly difficult decision to change the direction of a bullet in a platformer game. The solution is so complex that the brain was just boiling. And the desire to further study something disappears. It is too difficult to change the direction of a bullet in a platformer.

## Solution suggested
Make a separate normal bullet behavior, which depends on the rotation of the sprite, and fires only once, only when released from the sprite. That would not change the trajectory in flight.

**Comment by 4ian** [comment]
Making a "bullet" behavior should be easy using custom behavior - you could even do it and suggest it as an extension :)

**Comment by arthuro555** [comment]
I think there is already a bullet extension in the extension repository

**Comment by wireshark10** [comment]
Hi @4ian  can I have a look at this issue ?

**Comment by 4ian** [comment]
Yep feel free to try something, no need to ask me ;)
